src/ostree.cc

```diff
@@ -215,17 +242,5 @@ Json::Value OstreePackage::toEcuVersion(const Json::Value &custom) {
 
 OstreePackage OstreePackage::getEcu(const std::string &ecu_serial, const std::string &ostree_sysroot,
```

Comment:
Let's keep getEcu for now. We will need to separate Uptane notion of a package (or target in Uptane terms) from OstreePackage soon anyway.